### ISSUE TITLE
sign Factor.app

### ISSUE THREAD

**Issue by mrjbq7** [slavapestov/factor#1226]
Currently rejected by Gatekeeper:

```
$ spctl --assess --type execute Factor.app
Factor.app/: rejected
```

Should sign (using what certificate?):

```
$ codesign Factor.app -s MyCertificate
```

And verify:

```
$ codesign --verify Factor.app
```

**Comment by mrjbq7** [comment]
The new developer program is for both iOS and Mac.  Maybe @erg you can sign it for next release?

**Comment by erg** [comment]
Ok, it works! It's using my mac developer key.

Fixed in 9946eefc88a884e671f9460c90e8e5a832abd9c9.
